### PR TITLE
Controller metrics reported from openshift-controller-manager

### PR DESCRIPTION
**Pull request by wewang58**
Auto OCP-29780 and  [pass log](http://pastebin.test.redhat.com/882461)
@adambkaplan  @openshift/devexp-qe @jianzhangbjz please help to review it, thanks

### CODE REVIEW

**Comment by jianzhangbjz** [comment]
@wzheng1 Could you help review your team's PR? Please add `/lgtm` label if it looks good to you, thanks!

File: test/extended/builds/build_metrics.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+var _ = g.Describe("[sig-builds] Devexp", func() {
+	defer g.GinkgoRecover()
+	const (
+		ns       = "openshift-controller-manager"
+		hostport = "8443"
+	)
+	var (
+		oc = exutil.NewCLIWithoutNamespace("default")
+	)
+	g.It("controller metrics reported from openshift-controller-manager [Slow]", func() {
```
Comment (jianzhangbjz) [inline test/extended/builds/build_metrics.go]: @wewang58 Please follow the [test case naming rules](https://docs.google.com/document/d/1k761p65J0Ig81hwZWaw73QJV6SBa7kPxbjuTEBlBbF4/edit#heading=h.irspvef076fy), thanks!

Comment (wewang58) [reply test/extended/builds/build_metrics.go]: Let me see, thanks.

Comment (wewang58) [reply test/extended/builds/build_metrics.go]: @jianzhangbjz  Sorry for updating the pr late, already Updated "[sig-team] sub-team" label according docs. could you check if rules ok, thanks

**Comment by wewang58** [comment]
@wzheng1 @xiuwang Could you help to review the pr? @adambkaplan if you have time, can you help to check too, hope you can give advice too, thanks all.

File: test/extended/builds/build_metrics.go (hunk 1 above)
Comment (adambkaplan) [reply test/extended/builds/build_metrics.go]: @wewang58 this test is validating `DeploymentConfig`, which is owned by the Workloads team. Please update the title to be `[sig-workloads] DeploymentConfig`

cc @tnozicka

Comment (wewang58) [reply test/extended/builds/build_metrics.go]: @adambkaplan thanks reviewing it, @tnozicka  Updated title for "[sig-apps] Workloads"

Comment (xiuwang) [reply test/extended/builds/build_metrics.go]: @wewang58 It should follow naming rules too, such as g.It("Low-23670-Checking description info for csc CRD", func()

Comment (jianzhangbjz) [reply test/extended/builds/build_metrics.go]: @wewang58  Please see @xiuwang comments.

Comment (wewang58) [reply test/extended/builds/build_metrics.go]: @jianzhangbjz @xiuwang  Updated it part with "Medium-29780--xxx", and [pass log](http://pastebin.test.redhat.com/898872), please check again, thanks

Comment (wewang58) [reply test/extended/builds/build_metrics.go]: @xiuwang  when you have time, could you help to review it again, thanks

**Comment by wewang58** [comment]
@jianzhangbjz Could you help to merge the pr, thanks

File: test/extended/builds/build_metrics.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+var _ = g.Describe("[sig-apps] Workloads", func() {
+	defer g.GinkgoRecover()
+	const (
+		ns       = "openshift-controller-manager"
+		hostport = "8443"
+	)
+	var (
+		oc = exutil.NewCLIWithoutNamespace("default")
+	)
+	g.It("Medium-29780-Controller metrics reported from openshift-controller-manager[Serial]", func() {
```
Comment (kuiwang02) [inline test/extended/builds/build_metrics.go:23]: @wewang58 could the case be executed in parallel? i.e. remove Serial, thanks

File: test/extended/builds/build_metrics.go [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/openshift-tests/test/extended/util"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = g.Describe("[sig-apps] Workloads", func() {
```
Comment (kuiwang02) [inline test/extended/builds/build_metrics.go:14]: @wewang58 could you please move the case to openshift-test-private repo? there are some workloads cases in there. cc @zhouying7780

Comment (wewang58) [reply test/extended/builds/build_metrics.go:14]: @kuiwang02  Thanks reviewing it, will remove to openshift-test-private repo.

**Comment by kuiwang02** [comment]
it moves to private repo.

**Pull request closed without merging**